safety_controller:
  ros__parameters:
    scan_topic: "/scan"
    ackermann_cmd_topic: "/vesc/low_level/ackermann_cmd"
    safety_topic: "/vesc/low_level/input/safety"
    stopping_time: 0.4
    watchdog_period: 0.1

cone_detector:
  ros__parameters:
    line_following: False

parking_controller:
  ros__parameters:
    drive_topic: "/vesc/high_level/input/nav_0"
    parking_distance: 0.75
    max_distance_error: 0.08
    max_angle_error: 5.0
    max_velocity: 1.0
    min_velocity: 0.5
    Dp: 1.0
    Dd: 0.1
    Ap: 0.5
    Ad: 0.03
